<?xml version="1.0"?>
<launch>
 <param name="gps_meters_reset_topic" value="/landshark/gps_meters/reset"/>
 <param name="gps_topic" value="/radl/gps"/>
 <param name="gps_meters_topic" value="/landshark/gps_meters"/>
 <param name="goal_topic" value="/radl/goal"/>
 <param name="goal_meters_topic" value="/landshark/goal_meters"/>
 <param name="map_topic" value="/radl/map"/>
 <param name="map_meters_topic" value="/landshark/map_meters"/>
 <param name="origin_mode" value="automatic"/> <!-- or "manual" uses origin_lat, origin_lon-->
<!-- <param name="origin_lat" value="37.4571014863"/>-->
<!-- <param name="origin_lon" value="-122.173496991"/>-->
 <param name="dt" value = "0.1" /> 

 <node pkg="landshark_gps_meters" type="landshark_gps_meters_node" name="landshark_gps_meters_node" output="screen" />

</launch> 
